### ISSUE TITLE
Project Topic & Goal Brainstorm

### ISSUE THREAD

**Issue by philippaSteinberg** [philippaSteinberg/EEP153-Proj1#18]
What Countries/Topics/Questions do we want to explore?

**Comment by juliarunkle** [comment]
5: Subnational Malnutrition Database
11: Africa Development Indicators
16: Health Nutrition and Population Statistics
40: Population estimates and projections
88: Food Prices for Nutrition

Potentially Use (If We Expand to Wealth/Poverty)
39: Health Nutrition and Population Statistics by Wealth Quintile
59: Wealth Accounts

**Comment by philippaSteinberg** [comment]
Possible country categories: 
`AFE`  Africa Eastern and Southern
`AFW` Africa Western and Central

**Comment by philippaSteinberg** [comment]
Possible Questions: 
- [ ] Population trends compared between AFE costal and AFW
- [ ] How does being a AFE vs AFW impact [#C] changes in agricultural food supply to changes in population (visualize food import and export rates)
- [ ] How does being a AFE vs AFW impact correlate nutrition affordability (population density of people who can afford nutritious diet with dataset 88)

**Comment by vkeroles** [comment]
Possible Stretch Goal: 
Utilize Nunn (2008) and Nunn and Puga (2012):  In Africa, ruggedness/terrain interacted with historical slave trades such that being in a more rugged land protected Africans from the slave trades. Having less slave trades generates an additional benign effect in Africa and partially undoes the negative impacts of ruggedness we observe elsewhere.

For Visualization, 
Nunn 2008 Table 2 "Estimated Total Slave Exports Between 1400 and 1900 By Country" is a list of African countries with a reported number of slave exports for each. 
Possible Issues / Concerns
—> Establish a threshold for the two groups; high exports and low/no exports? Make our own group/region datasets using datasets for each individual country?
—> To draw conclusions on effect on population growth, do we combine these groups into one population set or still compare on a country level?
—> Can we make the assumption that they do not differ besides in slave exports for the sake of project, or do we need to create some kind of model to control between countries?

**Comment by juliarunkle** [comment]
Another comparison we could make:

NRS   Non-resource rich Sub-Saharan Africa countries
RRS   Resource rich Sub-Saharan Africa countries

**Comment by juliarunkle** [comment]
Regions That Work for Each Source

#5: `AFE`

**Comment by juliarunkle** [comment]
Regions That Work for Each Source

#5: `AFE``AFW``CAF``MEA``MNA``SSA``SSF``TMN``TSS``ZAF`
#11: `AFR``CAF``NAF``SSA``SSF``SXZ``XZN``ZAF`
#16: `AFE``AFW``CAF``MEA``MNA``SSA``SSF``TMN``TSS``ZAF`
#40: `AFE``AFW``CAF``MEA``MNA``SSA``SSF``TMN``TSS``ZAF`

#39 and #59 only have information for each individual country, the data is not sorted by region. All of the above sources also have data for each country. For #88, I couldn't find African regions or countries that work for this data set, let me know if anyone figures this out.

**Comment by oghomiep** [comment]
Was reflecting on what prof suggested on Tuesday 1/25 about defining our "theme" , and had some thoughts:
- It doesn't seem like AFE and AFW represents the coastal vs inland categorization we were looking for. Just by inspection, it looks like there was significant traffic in both Western and Central (AFW), but also Northern and Eastern Africa.
![africa_map](https://user-images.githubusercontent.com/112025671/214950643-e3fdbabb-d436-43d7-8999-9d7adbb73faf.png)
![image](https://user-images.githubusercontent.com/112025671/214950704-e048eca9-c3f7-4750-91ce-16d36551ed6d.png)
- Since categorization of countries seems to be the biggest stumbling block, perhaps we can instead try to "validate" what's presented in Victor's slave trade paper? i.e. We see which individual countries/regions (depending on the dataset) in Africa are the most and least well off (in terms of gdp, population growth, etc.) and compare it to what the paper finds. 
- Some advantages of focusing on the "best" and "worst" countries/regions 
     - We don't have to do 58 separate plots/analyses. We can just look at the top and bottom few. 
     - We can incorporate the slave trade idea, giving our project some ~personality~.
- TL;DR this is what I propose 
     - Once we're done with all the A level deliverables, we apply it to Africa as follows 
     - Find the most and least well off African countries/regions based on classic indicators (gdp, education, etc.) -- can google 
     - Apply the deliverables (dataframes, pyramids) to these countries/regions 
     - See how it compares to expectations 
     - And if we want we can also track changes in agricultural food supply for these best/worst countries/regions.

**Comment by vkeroles** [comment]
Thank you for looking into this! Is the second map showing the movement of slaves? If so, I can check with the info in Nunn's paper. She does include a table in which there are estimates of the slave exports of each country and there is clear difference between those that had lot of slaves taken to the US and ones where they are minimal. Perhaps we can see if we can split this list into 2/3 groups and see if they match with any regional grouping in the datasets we having access to. From there, we can compare these groups in terms of GDP, food supply, etc. 

Let me know what you think and I can look into this more once I have the coding for my deliverable down!

**Comment by oghomiep** [comment]
yep, just found it through a quick google search. and that idea sounds good to me.

**Comment by vkeroles** [comment]
Here is a link to the [World Bank](https://data.worldbank.org/country). Right hand column of webpage has all the groupings used in datasets. After choosing one, you can scroll down and see which countries are included in this chosen group.

**Comment by juliarunkle** [comment]
Hi everyone! Unfortunately, I will not be in class tomorrow, but I thought it's important that I update yall on what I've been working on before class so that you can continue with the project :)
I've been diving into the data sources we discussed (5, 11, 16, 39, 40, 59) and cross-referencing that data with the countries in Nunn's paper, and I hate to say it but I'm just not really finding correlations there that can relate to population/food supply. I've looked at the probability of death for various age groups, life expectancy, fertility rate, age distribution in the labor force, population growth, infant mortality, unemployment, malnutrition, access to clean drinking water, and more. I pulled from the countries with the highest number of slave trades and the lowest, and for some categories, I even went through every single country, and I just wasn't finding correlations.
I thought this was a really fascinating angle to take with this project, but unfortunately for our purposes, I don't think this research question is going to take us in the direction we wanted or were expecting.
Personally, I'd suggest comparing geographical regions, possibly debt, aid from other countries, or even just linking simple things like malnutrition/access to clean drinking water, or if someone could figure out the food prices source (88), I think those would be great directions.
Please let me know what yall decide, and I'm happy to do any research necessary. Good luck tomorrow!

**Comment by eddieguo-1128** [comment]
Current notebook is updated containing the two [#A] deliverables.

**Comment by vkeroles** [comment]
@eddieguo-1128 Hey, Can you update notebook with my [#B] deliverable and the update I made to the [#A] dataframe deliverable? Thank you!

**Comment by oghomiep** [comment]
Christine Chiang is inviting you to a scheduled Zoom meeting.

NOTE: All participants and hosts are now required to sign into a Zoom account prior to joining meetings hosted by UC Berkeley. See "How to sign into your UC Berkeley Zoom account" (https://berkeley.service-now.com/kb?id=kb_article_view&sysparm_article=KB0013718) for how to sign in.

Participants who are not eligible for a UC Berkeley-provided Zoom account can use a Zoom account provided by their institution, can create a free, consumer Zoom account (at https://zoom.us/freesignup/), or can dial in via the phone.


Topic: Christine Chiang's Personal Meeting Room

Join Zoom Meeting
https://berkeley.zoom.us/j/9098238528

Meeting ID: 909 823 8528
One tap mobile
+12133388477,,9098238528# US (Los Angeles)
+16692192599,,9098238528# US (San Jose)

Dial by your location
        +1 213 338 8477 US (Los Angeles)
        +1 669 219 2599 US (San Jose)
        +1 669 900 6833 US (San Jose)
        +1 206 337 9723 US (Seattle)
        +1 253 215 8782 US (Tacoma)
        +1 346 248 7799 US (Houston)
        +1 602 753 0140 US (Phoenix)
        +1 720 928 9299 US (Denver)
        +1 971 247 1195 US (Portland)
        +1 267 831 0333 US (Philadelphia)
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 470 250 9358 US (Atlanta)
        +1 470 381 2552 US (Atlanta)
        +1 646 518 9805 US (New York)
        +1 646 558 8656 US (New York)
        +1 651 372 8299 US (Minnesota)
        +1 786 635 1003 US (Miami)
Meeting ID: 909 823 8528
Find your local number: https://berkeley.zoom.us/u/aez3MYUU5

Join by SIP
9098238528@zoomcrc.com

Join by H.323
162.255.37.11 (US West)
162.255.36.11 (US East)
221.122.88.195 (China)
115.114.131.7 (India Mumbai)
115.114.115.7 (India Hyderabad)
213.19.144.110 (Amsterdam Netherlands)
213.244.140.110 (Germany)
103.122.166.55 (Australia Sydney)
103.122.167.55 (Australia Melbourne)
209.9.211.110 (Hong Kong SAR)
149.137.40.110 (Singapore)
64.211.144.160 (Brazil)
69.174.57.160 (Canada Toronto)
65.39.152.160 (Canada Vancouver)
207.226.132.110 (Japan Tokyo)
149.137.24.110 (Japan Osaka)
Meeting ID: 909 823 8528

**Comment by vkeroles** [comment]
FIle to know which countries are within World Bank groupings
[CLASS.xlsx](https://github.com/philippaSteinberg/EEP153-Proj1/files/10669990/CLASS.xlsx)
